Danki2/Assets/Scripts/UI/CooldownBar.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CooldownBar : MonoBehaviour
```

Comment:
May want a generic bar class, which these inherit from, as this script has a lot of similar code as the previous one.